doc/examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
The linker flags are needed, since CMake will do separate compile and link steps. As I recall, the workaround for older CMake versions is to add `gcov` to the `target_link_libraries()`. However, at that point you basically have to give the entire `CMakeLists.txt`.

If you feel this essential, I can convert.

Reply:
I've made the change, and put notes in the cookbook file about changes for newer CMake versions.